src/pcapng.h

```diff
@@ -290,3 +291,82 @@ PcapNgWriteEnhancedPacket(
 
     return Err;
 }
+
+inline int
+PcapNgWriteHciPacket(
+    HANDLE File,
+    char* FragBuf,
+    unsigned long FragLength,
+    unsigned long OrigFragLength,
+    long InterfaceId,
+    long IsSend,
+    unsigned char Type,
+    long TimeStampHigh,
+    long TimeStampLow
+)
+{
+    int Err = NO_ERROR;
+    struct PCAPNG_BLOCK_HEAD Head;
+    struct PCAPNG_ENHANCED_PACKET_BODY Body;
+    struct PCAPNG_BLOCK_TAIL Tail;
+    char Pad[4] = { 0 };
+    long Direction = htonl(!IsSend);
+    int FragPadLength = (4 - ((sizeof(Body) + sizeof(Direction) + sizeof(Type) + FragLength) & 3)) & 3; // pad to 4 bytes per the spec.
+    int TotalLength = sizeof(Head) + sizeof(Body) + sizeof(Direction) + sizeof(Type) + FragLength + FragPadLength + sizeof(Tail);
+
+    Head.Type = PCAPNG_BLOCKTYPE_ENHANCED_PACKET;
```

Comment:
This function is writing a block of type PCAPNG_BLOCKTYPE_ENHANCED_PACKET. But there's already a function for writing that type of block: PcapNgWriteEnhancedPacket. Why do we need the new function?